### ISSUE TITLE
Probes show up on one plane with 100 micrometer brain atlas

### ISSUE THREAD

**Issue by rcpeene** [AllenInstitute/openscope_databook#47]
I changed the brain atlas used by the neuropixel visualization notebook to the 100 micrometer atlas. In this depiction, the probes all appear on one plane, which is different than when using the other atlas. Further investigation is required.

**Comment by jeromelecoq** [comment]
@rcpeene to add example screenshots here.

**Comment by rcpeene** [comment]
I have been unsuccessful at attempts to reproduce this. It may have been a problem with the data. I don't have the file ID anymore but was an old file, so I will dismiss the issue.

**Comment by rcpeene** [comment]
Rediscovered this with visual coding data. Here are some captures.
![capture1](https://user-images.githubusercontent.com/23059582/200698430-32dcada7-7a9a-4cd7-b3fa-48fd2eda4b9c.PNG)
![Capture2](https://user-images.githubusercontent.com/23059582/200698433-7f43c855-76f9-4582-8186-0f8492bd275a.PNG)

**Comment by jeromelecoq** [comment]
@rcpeene to talk to @jsiegle about this. Perhaps it could be a problem in the NWB file itself.

**Comment by rcpeene** [comment]
`sub-699733573/sub-699733573_ses-715093703.nwb` is the file shown here.

**Comment by jsiegle** [comment]
Yes, this is a known issue with these NWB files. Unfortunately one set of axes values was duplicated at the time of packaging. However, if you access the data via the AllenSDK, the correct values will be read from a separate file.
